manifests/d/DuckStudio/FufuDevTools/2024.06.24.2036/DuckStudio.FufuDevTools.installer.yaml

```
@@ -27,4 +27,4 @@ Installers:
   InstallerUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/releases/download/202406242036/Fufu_Dev_Tools_Setup_202406242036.exe
```

```suggestion
  InstallerUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/releases/download/2024.06.24.2036/Fufu_Dev_Tools_Setup_2024.06.24.2036.exe
```

Sorry, forgot to add “.” to the tag when release this version.